### PR TITLE
Moved TestConstants.kt from core into test-utils

### PR DESCRIPTION
**Pull request by shamsasari**

### CODE REVIEW

File: node/src/test/java/net/corda/node/services/vault/VaultQueryJavaTests.java [hunk 1]
```diff
@@ -121,9 +119,9 @@ public void consumedCashStates() {
 
             Amount<Currency> amount = new Amount<>(100, Currency.getInstance("USD"));
 
-            fillWithSomeTestCash(services,
+            net.corda.testing.contracts.VaultFiller.fillWithSomeTestCash(services,
```
Comment (rnicoll) [inline node/src/test/java/net/corda/node/services/vault/VaultQueryJavaTests.java]: Could we avoid using full package name repeatedly throughout a file, it's rather spammy?

File: node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt [hunk 2]
```diff
@@ -800,28 +795,6 @@ abstract class AbstractNode(open val configuration: NodeConfiguration,
...
-            if (myIdentity.owningKey !is CompositeKey) { // TODO: Support case where owningKey is a composite key.
-                keyStore.save(serviceName, privateKeyAlias, keyPair)
-            }
-            val partyAndCertificate = getTestPartyAndCertificate(myIdentity)
-            // Sanity check the certificate and path
-            val validatorParameters = PKIXParameters(setOf(TrustAnchor(DUMMY_CA.certificate.cert, null)))
-            val validator = CertPathValidator.getInstance("PKIX")
-            validatorParameters.isRevocationEnabled = false
-            validator.validate(partyAndCertificate.certPath, validatorParameters) as PKIXCertPathValidatorResult
-            Pair(partyAndCertificate, keyPair)
```
Comment (rnicoll) [inline node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt]: Pretty certain this will break `BFTNotaryTests`, which depends on `ServiceIdentityGenerator`, which currently stores the keys in individual private & public key files. BFT notary tests seem to be disabled which is why we haven't seen this, investigating why we've got disabled tests, now.

File: node/src/test/java/net/corda/node/services/vault/VaultQueryJavaTests.java (hunk 1 above)
Comment (shamsasari) [inline node/src/test/java/net/corda/node/services/vault/VaultQueryJavaTests.java]: That's Intellij doing, I'll fix it.

File: node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt (hunk 2 above)
Comment (shamsasari) [inline node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt]: That may explain the failing tests...

**Comment by shamsasari** [comment]
Addressed comments, back to you.

File: node/src/test/java/net/corda/node/services/vault/VaultQueryJavaTests.java [hunk 3]
```diff
@@ -137,9 +137,9 @@ public void consumedCashStates() {
 
             Amount<Currency> amount = new Amount<>(100, Currency.getInstance("USD"));
 
-            fillWithSomeTestCash(services,
+            VaultFiller.fillWithSomeTestCash(services,
                                  new Amount<>(100, Currency.getInstance("USD")),
-                                 getDUMMY_NOTARY(),
+                                 net.corda.testing.TestConstants.getDUMMY_NOTARY(),
```
Comment (rnicoll) [inline node/src/test/java/net/corda/node/services/vault/VaultQueryJavaTests.java]: Full package names still present on `TestConstants`

File: test-utils/src/main/kotlin/net/corda/testing/contracts/VaultFiller.kt [hunk 4]
```diff
@@ -64,11 +64,11 @@ fun ServiceHub.fillWithSomeTestLinearStates(numberToCreate: Int,
         // Issue a Linear state
         val dummyIssue = TransactionType.General.Builder(notary = DUMMY_NOTARY).apply {
             addOutputState(DummyLinearContract.State(linearId = UniqueIdentifier(externalId),
-                                                     participants = participants.plus(me),
-                                                     linearString = linearString,
-                                                     linearNumber = linearNumber,
-                                                     linearBoolean = linearBoolean,
-                                                     linearTimestamp = linearTimestamp))
+                    participants = participants.plus(me),
```
Comment (rnicoll) [inline test-utils/src/main/kotlin/net/corda/testing/contracts/VaultFiller.kt]: Happy with either indenting a fixed amount or in-line with text above, but please document the decision in `codestyle.rst` (happy for this to be a separate PR)

Comment (shamsasari) [inline test-utils/src/main/kotlin/net/corda/testing/contracts/VaultFiller.kt]: Again this was Intellij auto-formatting when VaultFiller.kt was moved. I moved linearId to the next line so that it aligns - this is what I normally do anyway.

**Comment by shamsasari** [comment]
Comments addressed.
